modules/_UXTeam.scss

```diff
@@ -0,0 +1,113 @@
+@font-face {
+    font-family: Montserrat;
+    src: url(fonts/static/Outfit-ExtraLight.ttf);
```

Comment:
Another task - install Prettier as a plugin to format your code. See what hotkeys are for formatting your file AND configurate VS code to format your scss and html files on save with Prettier.